File: fabric/src/main/java/dev/gluton/quickspyglasser/fabric/QuickSpyglasserFabric.java
package dev.gluton.quickspyglasser.fabric;

import dev.gluton.quickspyglasser.QuickSpyglasser;
import net.fabricmc.api.ModInitializer;

public class QuickSpyglasserFabric implements ModInitializer {
    @Override
    public void onInitialize() {
        QuickSpyglasser.init();
    }
}

File: fabric/src/main/kotlin/dev/gluton/quickspyglasser/fabric/QuickSpyglasserFabric.kt
package dev.gluton.quickspyglasser.fabric

import dev.gluton.quickspyglasser.QuickSpyglasser
import net.fabricmc.api.ModInitializer

object QuickSpyglasserFabric : ModInitializer {
	override fun onInitialize() {
		QuickSpyglasser.init()
	}
}
